frontend/ui/widget/inputdialog.lua

```diff
@@ -424,7 +445,7 @@ function InputDialog:init()
         bordersize = self.border_size,
         background = Blitbuffer.COLOR_WHITE,
         VerticalGroup:new{
-            align = "left",
+            align = "center",
```

Comment:
Why?
Because you want your checkbutton centered ? If yes, may be wrap it in a CenterContainer like it's done for input_widget.
(Or remove that CenterContainer - but then, description is better kept left aligned - and may be it works because we have no bug when computing the width, but still, feels best to not mix the various ways of centering :)

Reply:
Yes, but I wait with this one until it gets clear if we want more checkboxex.